### ISSUE TITLE
Millisecond precision lost [JIRA: CLIENTS-749]

### ISSUE THREAD

**Issue by lukebakken** [basho/riak-python-client#430]
From a Riak TS user:

```
put : ['site1', 'species1', datetime.datetime(2016, 2, 1, 21, 6, 45, 45289), 22.5]
query : [['site1', 'species1', datetime.datetime(2016, 2, 1, 21, 6, 45), 22.5]]
```

**Comment by lukebakken** [comment]
Depends on #429

**Comment by lukebakken** [comment]
This bug is affecting the fix for Python 3.3 and 3.4: https://bugs.python.org/issue23517
